src/events/models/Attendee.ts

```diff
@@ -13,3 +15,14 @@ export interface IAttendeeUser {
   first_name: string;
   last_name: string;
 }
+
+// Attendee from the registration attendee list endpoint.
+export interface IUserAttendee {
+  id: number;
+  event: number;
+  user: IAttendeeUser;
+  attended: boolean;
+  timestamp: IsoDateTime;
+  show_as_attending_event: boolean;
+  has_paid: boolean;
+}
```

Comment:
This interface is super-similarly named to the interface above, might want to change things up a bit.